cordova-lib/src/plugman/registry/registry.js

```diff
@@ -218,10 +205,18 @@ module.exports = {
 
 /**
  * @method initSettings
+ * @param {Boolean} using npm registry
  * @return {Promise.<Object>} Promised settings.
  */
-function initSettings() {
+function initSettings(npm) {
     var settings = module.exports.settings;
+    var registryURL = 'http://registry.cordova.io';
+
+    //if npm is true, use npm registry
+    if(npm) {
+        registryURL = 'http://registry.npmjs.org';
+    }
+
```

Comment:
Minor nit - rather than setting registryURL to one value, then potentially setting it to another value, why not just:

    var registryURL = npm ? 'http://registry.npmjs.org' : 'http://registry.cordova.io';